webpack.config.js

```diff
@@ -27,7 +27,11 @@ const config = {
     filename: outputFile,
     library: libraryName,
     libraryTarget: 'umd',
-    umdNamedDefine: true
+    umdNamedDefine: true,
+
+    // This needs to be set to 'this' instead of the default 'self', since this package is accessed via the broser (script tag), or the server (npm install & require())
```

Comment:
`s/broser/browser/`

😉

Reply:
yw! just noticed this come across my notifications.  Doing great here, hope the same for you 🍻